modules/snooker/src/main/ts/ephox/snooker/resize/Bars.ts

```diff
@@ -12,6 +12,33 @@ const resizeRowBar = Styles.resolve('resizer-rows');
 const resizeColBar = Styles.resolve('resizer-cols');
 const BAR_THICKNESS = 7;
 
+const resizableRows = (warehouse: Warehouse, table: SugarElement<HTMLTableElement>, isResizable: (table: SugarElement<HTMLTableElement>, elm: SugarElement<Element>) => boolean): number[] => {
+  const rows = warehouse.all;
+  const resizableRows: number[] = [];
+  Arr.each(rows, (row, i) => {
```

Comment:
This should probably be using `Arr.bind` to avoid the mutation. eg:
```
return Arr.bind(rows, (row, i) => {
  return isResizable(table, row.element) ? [ i ] : [];
});

Reply:
Thanks! Fixed